src/components/learner-credit-management/BudgetDetailPageOverviewUtilization.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Stack, Collapsible, Row, Col, Button, Hyperlink} from '@edx/paragon';
+import {formatPrice, useBudgetId, useSubsidyAccessPolicy} from "./data";
+import { ArrowDownward } from '@edx/paragon/icons';
+import {useHistory} from "react-router";
+// import ge from "@faker-js/faker/locales/ge";
+import {generatePath, useRouteMatch} from "react-router-dom";
+
+const BudgetDetailPageOverviewUtilization = ({ budgetInfo: { utilized }, aggregates, isAssignable }) => {
+  const utilizedUsd = formatPrice(utilized);
+  const amountAllocatedUsd = aggregates.amountAllocatedUsd;
+  const amountRedeemedUsd = aggregates.amountRedeemedUsd;
+  const history = useHistory();
+  const routeMatch = useRouteMatch();
+  
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  
+  if (utilized <= 0 || !isAssignable) {
+    return null;
+  }
+  
+  const handleTableSkipLink = (type) => {
+    const location = {
+      pathname: generatePath(routeMatch.path, { budgetId: subsidyAccessPolicyId, activeTabKey: 'activity' }),
+      state: { budgetActivityScrollToKey: type }
+    }
+    
+    history.push(location);
+  }
+  
+  const renderActivityLink = ({amount, type}) => {
+    if (amount <= 0) {
+      return null;
+    }
+    
+    const linkText = (type === 'assigned') ? 'View assigned activity' : 'View spent activity';
+    
+    return (
+      <Button
+        onClick={() => handleTableSkipLink(type)}
+        size="sm" variant="link" iconAfter={ArrowDownward} className='pl-0 pr-0'>
+        {linkText}
+      </Button>
+    );
+  };
+  
+  return (
+    <Collapsible
+      className='mt-4 budget-utilization-container'
+      styling="basic"
+      title={<h6 className="mb-0">Utilization details</h6>}
+      defaultOpen
```

Comment:
The `Collapsible` should be not be opened by default.

<img width="667" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/9a8284d5-bb0a-403a-8583-741af4bf3ceb">